Makefile

```diff
@@ -1,10 +1,29 @@
 BUILD_TARGET=workload-app
 NAMESPACE=workload-web-app
+CONTAINER_ENGINE=docker
+TOOLS_IMAGE=quay.io/integreatly/workload-web-app-tools
```

Comment:
Has this image been built and pushed anywhere?
I guess we need to add this an extra image to build on CI also?